### DISCUSSION TITLE
Dark theme box shadow

### DISCUSSION THREAD

**Discussion by jdanyow** [microsoft/atlas-design#194] [Component]
Unnatural? Outrageous? Should we remove it? I don't think we really put much thought into it when we overhauled themes... hard to remember that far back 🤷‍♂️

I can't stop thinking about [this tweet](https://twitter.com/ryanflorence/status/1376023343992737798?s=20)

![image](https://user-images.githubusercontent.com/1306747/118739896-69880f00-b7ff-11eb-9946-8e76eb31eec8.png)

**Comment by heskew** [comment]
Wait, what? The slight glow at the edges of the boxes that looks like some light reflecting off the edges? Seems natural enough to not worry about. It's all about perspective.😏

**Comment by Courtny** [comment]
It's goofy. We should remove it. 

We also should spend some time _(in the future)_ doing an actual proper overhaul of our Dark Theme as a design system team. Check out the retrospective of the effort that went into [Dark Mode on Stack Overflow](https://stackoverflow.blog/2020/03/31/building-dark-mode-on-stack-overflow/). Much nuance and wow!

**Reply by heskew** [reply]
But then I won't feel like I'm in the early-mid 2000s 😢

> We also should spend some time (in the future) doing an actual proper overhaul of our Dark Theme as a design system team.

^^ yes please

**Reply by jdanyow** [reply]
Looking forward to that! We've done the work to abstract away the palette from the functional/semantic colors. Lot of flexibility to tailor things for specific themes like [high contrast](https://docs.microsoft.com/en-us/dotnet/core/introduction?theme=high-contrast).

**Comment by maschad96** [comment]
I vote for removing it. It feels like a default photoshop 'outer glow' filter, and the tweet is accurate 😂

**Reply by heskew** [reply]
don't worry. it's just light from your monitor escaping through the edges of the block elements. helps to keep the backlight from overheating...

**Reply by jdanyow** [reply]
it's box-sizing:content-box leaking out
